webapp/packages/plugin-data-viewer-result-set-grouping/src/DVGroupingColumnEditorDialog/GroupingColumnEditorTable.m.css

```diff
@@ -0,0 +1,23 @@
+.header {
+  composes: theme-border-color-background theme-background-surface theme-text-on-surface from global;
+  overflow: hidden;
+  display: flex;
+  flex-direction: column;
+  padding-bottom: 16px;
+  border-bottom: 1px solid;
+  gap: 16px;
+  flex: 1 0 auto;
+}
```

Comment:
maybe you can remove this element completely and use `<hr/>`

Reply:
I can't remove this element completely, but I can get rid of some properties using `<Container />` 
If using `<hr /> `I will have to style it separately, so I don't really see the point.